### ISSUE TITLE
Cancelling a failing import doesn't seem to do anything

### ISSUE THREAD

**Issue by ericzundel** [pantsbuild/intellij-pants-plugin#43]
Plugin version:  1.1.2
IntelliJ version:  14.0.3
Pants version:  square-20150331-02  https://github.com/square/pants/releases/tag/square-20150331-02

I was trying to reproduce an issue from a developer.  I tried importing a new project then went away from my laptop during the 'resolve' phase.  After 10 minutes it seemed stuck, so I hit the 'cancel' button.  Nothing changed on the UI and the process still seems to be running.

```
./pants compile multipass::
2015-04-28 07:22:23,011: Regenerated BUILD files in 4.232 seconds.
Running Pants version square-20150331-02
INFO] Detected git repository at /Users/zundel/Development/java on branch master

07:22:24 00:00 [main]
               (To run a reporting server: ./pants server)
07:22:24 00:00   [bootstrap]
07:22:25 00:01   [setup]
07:22:25 00:01     [parse]
               Executing tasks in goals: bootstrap -> imports -> unpack-jars -> deferred-sources -> gen -> resolve -> compile
Waiting on pants process 29260 (/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python /Users/zundel/.cache/pants/bin/pants/pants-square-20150331-02.pex goal resolve BUILD:) to complete
```

![screen shot 2015-04-28 at 7 19 45 am](https://cloud.githubusercontent.com/assets/119728/7368551/1b45d844-ed77-11e4-925d-bf6060eb041e.png)

From the command line, 

```
$ kill -TERM 29260
```
Make the process running on behalf of IntelliJ exit.

**Comment by fkorotkov** [comment]
Will be fixed with 1.2 release. Already in master dc590ddf6851eb23d5149efc2d21ad850cdb7c65
